libs/dextool/source/dextool/compilation_db/package.d

```diff
@@ -714,16 +727,25 @@ ParseFlags parseFlag(CompileCommand cmd, const CompileCommandFilter flag_filter,
             skip,
             /// skip the next arg, if it is not a flag
             skipIfNotFlag,
+	    /// use the next arg to create a complete path
+	    checkingForQuotation,
         }
 
-        import std.path : buildNormalizedPath, absolutePath;
+	import std.path : buildNormalizedPath, absolutePath;
         import std.array : appender, array;
         import std.range : ElementType;
 
         auto st = State.keep;
         auto rval = appender!(string[]);
         auto includes = appender!(string[]);
         auto compiler = Compiler(r.length == 0 ? null : r[0]);
+	auto path = appender!(string);
+
+	void putNormalizedAbsolute(string arg) {
```

Comment:
make it a function by adding `static`. Then we know at compile time that it doesn't use any local variables (e.g. is a delegate).